src/Microsoft.Health.Dicom.Functions/AzureFunctionsConfiguration.cs

```diff
@@ -0,0 +1,12 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Functions
+{
+    internal static class AzureFunctionsConfiguration
```

Comment:
do we need this to be static?

Reply:
This is meant to be a static class that holds some constants about configuration. So I don't mean for this to be instantiated. Perhaps there is a better name for this, but hopefully `AzureFunctionsConfiguration` is too generic for our own usage as a config object